plugins/osdn/plugin.go

```diff
@@ -173,8 +180,23 @@ func (plugin *OsdnNode) SetUpPod(namespace string, name string, id kubeletTypes.
 }
 
 func (plugin *OsdnNode) TearDownPod(namespace string, name string, id kubeletTypes.DockerID) error {
+	hostVethName, contMac, contIP, err := plugin.vethInspect(id)
+	if err != nil {
+		return err
+	}
+
+	containerNs, err := plugin.getContainerNS(id)
+	if err != nil {
+		return fmt.Errorf("Failed to get container network namespace: %v", err)
+	}
+	defer containerNs.Close()
+
+	if _, err := plugin.runIPAM(containerNs, "DEL", id); err != nil {
+		return err
+	}
+
 	// The script's teardown functionality doesn't need the VNID
-	out, err := exec.Command(plugin.getExecutable(), tearDownCmd, string(id), "-1", "-1", "-1").CombinedOutput()
+	out, err := exec.Command(plugin.getExecutable(), tearDownCmd, hostVethName, contMac, contIP, "-1", "", "", "false").CombinedOutput()
```

Comment:
note that in addition to the dummy arguments that are already ""/"-1", hostVethName and contMac are also unused here. Maybe we should just make setup and teardown take different args? (But maybe this will all eventually go away and it doesn't matter.)